src/services/identity/unauthorized.ts

```diff
@@ -1,83 +1,59 @@
-import { LocalIdentity } from './local.js';
-import type { VerifiableCredential, IVerifyResult, VerifiablePresentation } from '@veramo/core';
-import { MemoryPrivateKeyStore } from '@veramo/key-manager';
-import { KeyManagementSystem } from '@veramo/kms-local';
+import { MemoryPrivateKeyStore } from '@veramo/key-manager'
```

Comment:
Bad merge?

Reply:
It is not a bad merge.

Currently, in develop branch we don't have src/services/identity/IIdenity.ts because this file context will be moved in src/services/identity/IIdentity.ts.

I created src/services/identity/IIdenity.ts file for fixing problem with cycle import.
I refactored code in IIdentity.
we have identical same methods: 6 methods + a lot of not implemented methods in Local and Unauthorized.
- resolveDid (return DIDDocument = DID Resolution), resolve (support query parameters = Dereferencing), verifyCredential, verifyPresentation, checkStatusList2021, searchStatusList2021 - same methods in different subclasses such as:
- Unauthorized, Postgres, Local.
For that reason I decided to refactor code. In currently design looks like that:
1. IIdentity (interface) - provide all methods which should be implemented.
2. AbstractIdentity (abstract class) - which implement all IIdentity methods like not impelemented.
3. DefaultIdentity (class) - which extend AbstractIdentity and implement only 6 base methods which will be same for subclasses.
4. Postgres, Local, Unauthorized (class) - which extend DefaultIdentity and implement specific methods which will be supported for these classes.